assets/lang/pl_PL.ini

```diff
@@ -18,20 +18,20 @@ Use global volume = Używaj głośności globalnej
 WASAPI (fast) = WASAPI (szybki)
 
 [Controls]
-Analog Binding = Analog Binding
-Analog Limiter = Ogranicznik analoga
-Analog Settings = Analog Settings
+Analog Binding = Połączenie Gałki Analogowej
```

Comment:
Binding to bardziej "przypisanie" niż "połączenie", chodzi w końcu o przypisanie przycisków/kierunków a nie o połączenie ich.

W mowie potocznej nikt, przynajmniej żaden gracz również nie mówi "Gałka Analogowa", śmiesznie to brzmi.

Reply:
> W mowie potocznej nikt, przynajmniej żaden gracz również nie mówi "Gałka Analogowa", śmiesznie to brzmi.

Wiem, ale nie mam lepszego pomysłu :smile: Może `Joystick`? "Przypisanie JoyStick'a"?

Reply:
Myślę, że wystarczyłoby "Przypisanie Analoga" i pozmieniać też dotychczasowe tłumaczenia, często mniej znaczy więcej a na pewno przejrzyściej.

Reply:
okej: ed3a30aa1
Problem w tym, że samo "Analog" jako tłumaczenie do "Analog Stick" również nie brzmi moim zdaniem dobrze, więc:
- w wyrażeniach typu "Ogranicznik Analoga" jest to imo og
- ale "Analog Stick = Gałka Analogowa"; "Right Analog Stick = Prawa Gałka Analogowa"
ok?